src/core/keychain/keychainTypes/hardwareWalletKeychain.ts

```diff
@@ -10,8 +10,10 @@ import { HWSigner } from '../HWSigner';
 import { IKeychain, PrivateKey } from '../IKeychain';
 import { getHDPathForVendorAndType } from '../hdPath';
 
+export type HardwareWalletVendor = 'Ledger' | 'Trezor';
+
 export interface SerializedHardwareWalletKeychain {
-  vendor?: string;
+  vendor?: HardwareWalletVendor;
```

Comment:
Should this be an enum? We have a few instances where we use 'Ledger' and 'Trezor' directly